pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -540,7 +546,28 @@ func BootstrapWorkloadPinningMC(role string, pinningMode *apiconfigv1.CPUPartiti
 	}
 
 	mode := 420
-	source := "data:text/plain;charset=utf-8;base64,ewogICJtYW5hZ2VtZW50IjogewogICAgImNwdXNldCI6ICIiCiAgfQp9Cg=="
+	/*
+		Contents for emptyKubeletConfig:
+		{
+			"management": {
+				"cpuset": ""
+			}
+		}
+
+		Contents for emptyCrioConfig:
+		[crio.runtime.workloads.management]
+		activation_annotation = "target.workload.openshift.io/management"
+		annotation_prefix = "resources.workload.openshift.io"
+		resources = { "cpushares" = 0, "cpuset" = "" }
+	*/
+	emptyKubeletConfig := "ewogICJtYW5hZ2VtZW50IjogewogICAgImNwdXNldCI6ICIiCiAgfQp9Cg=="
+	emptyCrioConfig := "W2NyaW8ucnVudGltZS53b3JrbG9hZHMubWFuYWdlbWVudF0KYWN0aXZhdGlvbl9hbm5vdGF0aW9uID0gInRhcmdldC53b3JrbG9hZC5vcGVuc2hpZnQuaW8vbWFuYWdlbWVudCIKYW5ub3RhdGlvbl9wcmVmaXggPSAicmVzb3VyY2VzLndvcmtsb2FkLm9wZW5zaGlmdC5pbyIKcmVzb3VyY2VzID0geyAiY3B1c2hhcmVzIiA9IDAsICJjcHVzZXQiID0gIiIgfQo="
```

Comment:
I'm fine with the concept of this change but hardcoding the base64 encoded value seems unnecessarily obfuscated. I'd hardcoded the plaintext and base64 programmatically.
cc @MarSik @Tal-or

Reply:
Sure thing, updated to use our current flow for empty states